spec/unit/redirects_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe 'Redirects' do
+
+  before do
+    @redirects = Redirects.new
```

Comment:
You should probably hardcode your access token in this spec so that VCR sees the same request URL regardless of the environment.